README.md

```diff
@@ -642,6 +642,19 @@ can be used to opt out of hoisting for certain dependencies.
 $ lerna bootstrap --hoist --nohoist=babel-*
 ```
 
+#### --registry [registry]
+
+```sh
+$ lerna publish --registry https://my-private-registry
+$ lerna bootstrap --registry https://my-private-registry
+```
+
+When run with this flag, the above commands will use the specified registry for your package(s).
```

Comment:
I checked it, and found that the `run` command does not get decorated yet. Could you guys give me some wisdom on this? Do we need it to be present on run-s as well? I'm not sure about it.